examples/morphology/main.go

```diff
@@ -0,0 +1,35 @@
+// Port of http://members.shaw.ca/el.supremo/MagickWand/grayscale.htm to Go
+package main
+
+/*
+#cgo CFLAGS: -I/usr/local/include/ImageMagick-6
+#cgo LDFLAGS: -L/usr/sbin/lib
```

Comment:
Are these cgo directives just to get it to work relative to your own local dev machine? None of the other examples have them since the default behaviour is to rely on pkg-config.